Scripts/AirPlayer.gd

```diff
@@ -6,7 +6,12 @@ func _ready():
 	default_speed = 165
 	default_coyote_time = INF
 	
-func djump():
-	move_cooldown = default_move_cooldown
-	if is_on_floor() == false:
-		jump()
+var hasdjump: bool = true
+
+func jump() -> void:
+	velocity.y = max(velocity.y, 0) + 15
```

Comment:
Please use the `default_jump_strength` variable (defined in Player) instead of 15.

Reply:
`default_jump_strength` doesn't exist

Reply:
`git pull` my guy